### PR TITLE
speedup builds for core CI images

### PR DESCRIPTION
**Pull request by BenTheElder**
a fix was merged for these images an hour ago and images are still not ready

let's give these a bit more oomph

### CODE REVIEW

File: images/krte/cloudbuild.yaml [hunk 1]
```diff
@@ -28,6 +28,8 @@ substitutions:
   _CFSSL_VERSION: R1.2
 options:
   substitution_option: ALLOW_LOOSE
+  # this is a large and critical CI image, builds are slow on the default 1 core
+  machineType: N1_HIGHCPU_32
```
Comment (BenTheElder) [inline images/krte/cloudbuild.yaml:32]: this takes longer to startup (there's a pre warmed pool of default machines) but is *much* more powerful

Comment (BenTheElder) [reply images/krte/cloudbuild.yaml:32]: https://cloud.google.com/build/docs/build-config-file-schema#options

**Comment by cici37** [comment]
/lgtm
Thank you! (not sure if it's gonna add pressure on infra budget though :) )

**Comment by BenTheElder** [comment]
i'm going to switch to e2 type

File: images/krte/cloudbuild.yaml (hunk 1 above)
Comment (BenTheElder) [reply images/krte/cloudbuild.yaml]: https://github.com/kubernetes/ingress-nginx/issues/9813 and switched to e2 here

**Comment by BenTheElder** [comment]
/hold cancel
we're paying for the compute duration either way

**Pull request merged**